src/libraries/Microsoft.PowerFx.Json/Public/Types/UserProvidedTypeDefinitions/FormulaTypeToSchemaHelper.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Runtime.InteropServices;
```

Comment:
added by VS? 

You can turn off:
[https://stackoverflow.com/a/74163552/534514](https://stackoverflow.com/a/74163552/534514)